reference.md

```diff
@@ -664,33 +662,49 @@ Field Name | Presence | Type | Description
+`brand_id` | REQUIRED | Array | Array of service brand Ids defined in `service_brands.json`. Used to filter out the results the wait time is requested for. At least one service brand Id must be provided.
 
 The following fields are all attributes within the main "data" object for this query response.
 
 Field Name | Presence | Type | Description
 ---|---|---|---
 `wait_times` | REQUIRED | Array | An array that contains one object per `brand_id`
 \-&nbsp; `brand_id` | REQUIRED | ID | ID from a service brand defined in `service_brands.json`
-\-&nbsp; `wait_time` | REQUIRED | Non-negative Integer | Wait time in seconds the rider will need to wait in the location before pickup. 
+\-&nbsp; `estimated_wait_time` | REQUIRED | Non-negative Integer | The estimated wait time in seconds the rider will need to wait in the location before pickup.
+\-&nbsp; `estimated_travel_time` | OPTIONAL | Non-negative Integer | The estimated travel time in seconds from the pickup to dropoff location. Cannot be provided if a drop off location is not provided.
+\-&nbsp; `estimated_travel_cost` | OPTIONAL | Non-negative currency amount | The estimated fare cost of the trip from the pickup to dropoff location. Cannot be provided if a drop off location is not provided.
+\-&nbsp; `estimated_travel_cost_currency` | Conditionally REQUIRED | Currency code | Currency of the `estimated_travel_cost`. REQUIRED if `estimated_travel_cost` is provided.
+\-&nbsp; `booking_details` | OPTIONAL | Object | Optionally, an object with real time booking details can be provided.
+-&nbsp;\-&nbsp; `service_name` | OPTIONAL | String | If the service name needs to change due to real time booking changes, `service_name` can be provided to update the name of the on-demand service system to be displayed to the riders.
+-&nbsp;\-&nbsp; `android_url` | Conditionally REQUIRED | URL | Android App Links that can open the booking app on Android. At least of one `android_url`, `ios_url`, `web_url`, or `phone_number` needs to be provided.
+-&nbsp;\-&nbsp; `ios_url` | Conditionally REQUIRED | URL | iOS Universal Links that can open the booking app on iOS. At least of one `android_url`, `ios_url`, `web_url`, or `phone_number` needs to be provided.
+-&nbsp;\-&nbsp; `web_url` | Conditionally REQUIRED | URL | Web url to browse to in order to make the booking request. At least of one `android_url`, `ios_url`, `web_url`, or `phone_number` needs to be provided.
+-&nbsp;\-&nbsp; `phone_number` | Conditionally REQUIRED | Phone Number | Phone number to call to make the booking request. At least of one `android_url`, `ios_url`, `web_url`, or `phone_number` needs to be provided.
 
 ##### Examples:
 
-###### Query: 
+###### Query:
 
-`https://www.example.com/gofs/1/en/wait_time?pickup_lat=45.60&pickup_lon=-73.30&brand_id=regular`
+POST `https://www.example.com/gofs/1/en/wait_time`
 
-###### Response: 
+```jsonc
+{
+  "pickup_lat": 45.60,
+  "pickup_lon": -73.30,
+}
```

Comment:
The example doesn't conform to the proposed changes (missing required `brand_id`)
```suggestion
{
  "pickup_lat": 45.60,
  "pickup_lon": -73.30,
  "brand_id": ["regular_ride", "large_ride", "shared_ride"],
}
```

Reply:
updated